intf/vernacexpr.ml

```diff
@@ -299,11 +299,7 @@ type module_binder = bool option * lident list * module_ast_inl
 
 (** Cumulativity can be set globally, locally or unset locally and it
    can not enabled at all. *)
-type cumulative_inductive_parsing_flag =
-  | GlobalCumulativity
-  | GlobalNonCumulativity
-  | LocalCumulativity
-  | LocalNonCumulativity
+type cumulative_inductive_parsing_flag = bool option
```

Comment:
Can you please also update the comment, and explain a bit the meanings of the three states of this “boolean”.

Reply:
Comment updated.